docs/data/sdks/typescript-browser/index.md

```diff
@@ -294,7 +296,7 @@ amplitude.init(API_KEY, OPTIONAL_USER_ID, {
 
 --8<-- "includes/sdk-ts-browser/shared-plugins.md"
 
-##### Web attribution enrichment plugin
+#### Web attribution enrichment plugin
```

Comment:
I feel we shouldn't change this. 
This the structure of shared-plugins:
#### Plugin examples
##### Destination type plugin
##### Enrichment type plugin
This should either be:
##### Web attribution enrichment plugin
or ###### Web attribution enrichment plugin

Reply:
Looks like there is no `#### Plugin examples`. But the structure would be

### Plugins
   #### Web attribution enrichment plugin
       ##### Differences from base SDK
   #### Page view enrichment plugin
       ##### Differences from base SDK

Thanks for calling out. Deleted a # for the later two

Reply:
I was wrong. Plugin examples is embedded at a shared file.

Reply:
Reverted it